pages/common/q.md

```diff
@@ -0,0 +1,27 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file:
+
+`q -d, "SELECT * from {{path/to/file}}"`
```

Comment:
Ideally the example description should describe what the command does, as if translating its options and parameters to natural language (see [guideline n.3](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines)).